### ISSUE TITLE
Filter for free food

### ISSUE THREAD

**Issue by tmcheung** [AlfHou/Bedpres-Signup#1]
Need to be able to filter out the ones that don't serve food.
Very important feature!

**Comment by HaiyueChen** [comment]
I AGREE, I DONT HAVE TIME FOR COMPANIES THAT DONT SERVE ME FREE FOOD
